src/clj_kondo/impl/config.clj

```diff
@@ -47,6 +47,7 @@
                                :exclude-destructured-keys-in-fn-args false
                                :exclude-destructured-as false
                                :exclude-defmulti-args false}
+              :shadowed-fn-param-name {:level :warning}
```

Comment:
Looks like this should be `:shadowed-fn-param`.

Reply:
You're right, I'll fix it